test/ConfigAggregatorTest.php

```diff
@@ -230,6 +230,14 @@ public function testConfigAggregatorSetsHandlesUnwritableCache(): void
         self::assertFileDoesNotExist($this->cacheFile);
     }
 
+    public function testConfigAggregatorDoesNotLoadConfigFromCacheIfCacheFileIsEmpty(): void
+    {
+        file_put_contents($this->cacheFile, '');
+        $aggregator = new ConfigAggregator([], $this->cacheFile);
+
+        self::assertEmpty($aggregator->getMergedConfig());
```

Comment:
shouldn't this assert an array that matches the input configuration? i.e. `self::assertSame([], $aggregator->getMergedConfig());` - also, when the cache file is empty, do we expect the cache file to be updated with the now cached config?

Reply:
Good question, @gsteel. I'll take another look tonight.